benchmarks/bm_runner.py

```diff
@@ -537,6 +537,66 @@ def func(args: argparse.Namespace) -> None:
         _subprocess_runner([args.asv_sub_command, *args.asv_args], asv=True)
 
 
+class TrialRun(_SubParserGenerator):
+    name = "trialrun"
+    description = (
+        "Fast trial-run a given benchmark, to check it works : "
+        "in a provided or latest-lockfile environment, "
+        "with no repeats for accuracy of measurement."
+    )
+    epilog = (
+        "e.g. python bm_runner.py trialrun "
+        "MyBenchmarks.time_calc /tmp/testpython/bin/python"
+        "\n  NOTE: setting $DATA_GEN_PYTHON is equivalent to the "
+        "'runpath' argument."
+    )
+
+    def add_arguments(self) -> None:
+        self.subparser.add_argument(
+            "benchmark",
+            type=str,
+            help=(
+                "A benchmark name, possibly including wildcards, "
+                "as supported by the ASV '--benchmark' argument."
+            ),
+        )
+        self.subparser.add_argument(
+            "runpath",
+            type=str,
+            nargs="?",
+            help=(
+                "A path to an existing python environment, "
```

Comment:
Is this more accurate?

```suggestion
                "A path to an existing python executable, "
```